lib/EphemeralSocket.js

```diff
@@ -133,7 +133,9 @@ ephemeralSocket.prototype._writeToSocket = function (data) {
             this.options.host;
 
         this._socket.send(buf, 0, buf.length,
-            this.options.port, host);
+            this.options.port, host, cb);
+    } else {
+        cb(new Error('Not sending buffered data'));
```

Comment:
Let's change this error message.

`new Error('could not send(). The dgram socket is full. Ignoring buffer')`